performance_curves/performance_curve_bundle.py

```diff
@@ -0,0 +1,54 @@
+from performance_curves.performance_curve import *
+
+
+class PerformanceCurveBundle:
+    def __init__(self,
+                 y_true: np.ndarray,
+                 y_scores: List[np.ndarray],
+                 metric: Metric,
+                 model_names: List[str],
+                 num_bins: Optional[int] = None,
+                 order_descending: bool = True,
+                 random_model: bool = True,
+                 random_seed: Optional[int] = None,
```

Comment:
Users can set the seed in their own code. I don't think there is much value in controlling it ourselves. 

This way, the user's code won't read like `bundle.bundle["whaterver"]` which is confusing.

Reply:
We discussed this offline, but wanted to keep a record here just in case. It seems like `scikit-learn` uses `np.random.seed()` generator and ignores seed set by the built-in `random.seed()` ([reference](https://stackoverflow.com/questions/31057197/should-i-use-random-seed-or-numpy-random-seed-to-control-random-number-gener)). I removed the `random_seed` argument in our `RandomPerformanceCurve`, but it might be worth it to leave a note in our document saying that users should set seed using `numpy` instead of built-in `random`.